0x/Contracts/Exchange.cs

```diff
@@ -0,0 +1,151 @@
+﻿
+using Nethereum.Contracts;
+using Nethereum.Hex.HexTypes;
+using Nethereum.RPC.Eth.DTOs;
+using Nethereum.Web3;
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Threading.Tasks;
+using ZeroX.Enums;
+using ZeroX.Orders;
+using ZeroX.Utilities;
+using Transaction = ZeroX.Transactions.Transaction;
+using Nethereum.Hex.HexConvertors.Extensions;
+
+namespace ZeroX.Contracts
+{
+    public class Exchange : SmartContract
```

Comment:
`ExchangeContract` is more adequate in this case.

Reply:
Ok